### DISCUSSION TITLE
Does Trilium have a remote API to add/retrieve notes?

### DISCUSSION THREAD

**Discussion by HubKing** [TriliumNext/Trilium#2299] [Q&A] [answered]
All I see in the documentation is scripting API. But what I wonder is the API to get/add notes remotely. For example, that kind of API would make it easy to create a desktop UI application. One could still make such a programme even if there is no such API by mimicking the HTTP communication between a web browser and Trilium, but that would be not as convenient.

**Comment by zadam** [comment, marked as answer]
Hi, there's [custom request handler](https://github.com/zadam/trilium/wiki/Custom-request-handler) using which you can create a custom API.

But otherwise Trilium doesn't yet have a remote API, although it is planned.
